### ISSUE TITLE
Guzzle is locked on a very old version

### ISSUE THREAD

**Issue by SamGoody** [onlineconvert/onlineconvert-api-sdk-php#53]
Guzzlehttp is past version 7, and is installed by many other packages.
By locking it to version 6.4.1, you prevent this API from being installed.

The API for guzzle has not changed much, please test if it can be made compat with v7, and if so, update the requirements.

**Comment by SamGoody** [comment]
Your requirements could not be resolved to an installable set of packages.
```
  Problem 1
    - The requested package guzzlehttp/guzzle (locked at 6.4.1, required as ^7.0) is satisfiable by guzzlehttp/guzzle[6.4.1] but these conflict with your requirements or minimum-stability.
  Problem 2
    - Installation request for qaamgo/onlineconvert-api-sdk ^3.0 -> satisfiable by qaamgo/onlineconvert-api-sdk[3.0].
    - qaamgo/onlineconvert-api-sdk 3.0 requires guzzlehttp/guzzle ^6.2 -> satisfiable by guzzlehttp/guzzle[6.2.0, 6.2.1, 6.2.2, 6.2.3, 6.3.0, 6.3.1, 6.3.2, 6.3.3, 6.4.0, 6.4.1, 6.5.0, 6.5.1, 6.5.2, 6.5.3, 6.5.4, 6.5.5, 6.5.x-dev] but these conflict with your requirements or minimum-stability.
  Problem 3
    - codeception/codeception 2.5.6 requires guzzlehttp/guzzle >=4.1.4 <7.0 -> satisfiable by guzzlehttp/guzzle[4.1.4, 4.1.5, 4.1.6, 4.1.7, 4.1.8, 4.2.0, 4.2.1, 4.2.2, 4.2.3, 4.2.4, 4.x-dev, 5.0.0, 5.0.1, 5.0.2, 5.0.3, 5.1.0, 5.2.0, 5.3.0, 5.3.1, 5.3.2, 5.3.3, 5.3.4, 5.3.x-dev, 6.0.0, 6.0.1, 6.0.2, 6.1.0, 6.1.1, 6.2.0, 6.2.1, 6.2.2, 6.2.3, 6.3.0, 6.3.1, 6.3.2, 6.3.3, 6.4.0, 6.4.1, 6.5.0, 6.5.1, 6.5.2, 6.5.3, 6.5.4, 6.5.5, 6.5.x-dev] but these conflict with your requirements or minimum-stability.
    - codeception/codeception 2.5.6 requires guzzlehttp/guzzle >=4.1.4 <7.0 -> satisfiable by guzzlehttp/guzzle[4.1.4, 4.1.5, 4.1.6, 4.1.7, 4.1.8, 4.2.0, 4.2.1, 4.2.2, 4.2.3, 4.2.4, 4.x-dev, 5.0.0, 5.0.1, 5.0.2, 5.0.3, 5.1.0, 5.2.0, 5.3.0, 5.3.1, 5.3.2, 5.3.3, 5.3.4, 5.3.x-dev, 6.0.0, 6.0.1, 6.0.2, 6.1.0, 6.1.1, 6.2.0, 6.2.1, 6.2.2, 6.2.3, 6.3.0, 6.3.1, 6.3.2, 6.3.3, 6.4.0, 6.4.1, 6.5.0, 6.5.1, 6.5.2, 6.5.3, 6.5.4, 6.5.5, 6.5.x-dev] but these conflict with your requirements or minimum-stability.
    - codeception/codeception 2.5.6 requires guzzlehttp/guzzle >=4.1.4 <7.0 -> satisfiable by guzzlehttp/guzzle[4.1.4, 4.1.5, 4.1.6, 4.1.7, 4.1.8, 4.2.0, 4.2.1, 4.2.2, 4.2.3, 4.2.4, 4.x-dev, 5.0.0, 5.0.1, 5.0.2, 5.0.3, 5.1.0, 5.2.0, 5.3.0, 5.3.1, 5.3.2, 5.3.3, 5.3.4, 5.3.x-dev, 6.0.0, 6.0.1, 6.0.2, 6.1.0, 6.1.1, 6.2.0, 6.2.1, 6.2.2, 6.2.3, 6.3.0, 6.3.1, 6.3.2, 6.3.3, 6.4.0, 6.4.1, 6.5.0, 6.5.1, 6.5.2, 6.5.3, 6.5.4, 6.5.5, 6.5.x-dev] but these conflict with your requirements or minimum-stability.
    - Installation request for codeception/codeception (locked at 2.5.6) -> satisfiable by codeception/codeception[2.5.6].
```

**Comment by online-convert** [comment]
Thank you for reporting this.
We have released a new version 3.1 supporting guzzle version 7. This should solve your issue.
